ifrs17/Test/QueriesTest.ipynb

```diff
@@ -528,7 +528,7 @@
             "cell_type": "code",
             "source": [
                 "public async Task<ActivityLog> CheckLoadYieldCurveAsync(Args args, IEnumerable<YieldCurve> testData, ",
-                "\n                                           (int year , int month) dataNodeYearAndMonth, ",
+                "\n                                           (int year , int month, string yieldCurveName, string valuationApproach) dataNodeParameters, ",
```

Comment:
rename : parameters have already a meaning and are something else. Used dataNodeTestData

Reply:
Renamed dataNodeParameters to dataNodeTestData.